distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -1440,35 +1445,33 @@ public String sendNotPermittedRequest(String jolokiaEndpoint) {
 
   @Test
   public void testAccessGroupsGuest() throws Exception {
```

Comment:
These two tests weren't working as intended and wasn't configuring the system properly to actually test what it was intended to. It should be resolved now